content/issues/2021-06-04-postgresql-12.md

```diff
@@ -5,7 +5,6 @@ resolved: true
 resolvedWhen: 2021-06-04 09:10:40+00:00
 severity: down
 affected:
-  - API
```

Comment:
Is this true? (API does not work without database.)

Reply:
well, the pod was running all the time and UptimeRobot also doesn't claim any outage at that time, so just to be consistent to not confuse users of the status page

Reply:
OK. Thanks for clarification.